src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -368,29 +366,54 @@ export default defineConfig({
 
 ### Generating pages
 
+To create a route for a specific `page`, you can fetch its directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages` to render the `home` page:
+
+```astro title="src/pages/index.astro" {7,8,15} 
+---
+import { useStoryblokApi } from '@storyblok/astro'
+import BaseLayout from '../layouts/BaseLayout.astro'
+import Page from '../storyblok/Page.astro';
+
+const storyblokApi = useStoryblokApi();
+const { data } = await storyblokApi.get('cdn/stories/home');
+const content = data.story.content;
+---
+<html lang="en">
+  <head>
+    <title>Storyblok & Astro</title>
+  </head>
+  <body>
+    <Page blok={content} />
```

Comment:
```suggestion
    <Page blok={content} />
```

I am not sure why we are not using the Storyblok component (from the storyblok library) in this case. It should work as it was but only fetching data from one story.

Reply:
What is `Page` for then? I thought `Page` was for rendering the `page` type

Reply:
@Jutanium I see where this is coming from. Uhmm I am not sure which one should be used. Page is the container for multiple bloks in the page content-type story.